package.json

```diff
@@ -55,7 +54,8 @@
     "@types/lodash": "4.14.150",
     "@types/node": "10.17.20",
     "@types/prettier": "1.19.1",
-    "@types/react": "16.9.36",
+    "@types/react": "16.14.52",
+    "@types/scheduler": "0.16.8",
```

Comment:
our react types version didn't match UI

I thought this was the cause of the `className` issue because the `yarn ignore` script wasn't writing an ignore while UI marked it as error -- it wasn't, i ended up fixing that another way, but it seems correct to keep these versions the same

i had to pin a version for `scheduler` because [types react](https://www.npmjs.com/package/@types/react/v/16.14.52?activeTab=code) just asks for version `"*"` and the version being installed was missing something that the react types reference